.eslintrc.json

```diff
@@ -155,13 +155,7 @@
     "default-case": "error",
     "dot-notation": "error",
     "eol-last": "error",
-    "eqeqeq": [
-      "error",
-      "always",
-      {
-        "null": "ignore"
```

Comment:
how come we're removing this rule, it's there for quite a specific reason - of course, it's always subjective but I think we've not captured this change and it would cause a lot of breaking changes

Reply:
It was requested in this comment: https://github.com/NewOrbit/eslint-config/pull/249#discussion_r523937490 by Bart.